site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Storage and Cookies"
+seoTitle: "Chrome Extensions storage and cookies"
+date: 2023-08-01
+description: Overview of how web storage APIs and cookies work in extensions
+---
+
+Extensions can store cookies and access web storage APIs similar to a normal website. However, in
+some cases these behave differently in extensions.
+
+This article does not cover extension specific APIs. For more on these, see
+[chrome.storage][chrome-storage-api].
+
+## Storage {: #storage }
+
+### Persistence {: #storage-persistence }
+
+Extension storage is always persistent. For example, when a user uses the "Clear Browsing Data"
+feature, this does not affect extensions. This includes any data stored using web storage APIs (such
+as Local Storage and IndexedDB). Additionally, extensions are not subject to storage eviction under
+heavy memory pressure.
+
+By default, extensions are subject to the normal quota restrictions on storage, which can be checked
+using the `navigator.storage.estimate()` method. However, they can request the `unlimitedStorage`
+permission which affects both extension and web storage APIs.
+
+### Access in service workers {: #storage-in-service-workers }
+
+Some web storage APIs like IndexedDB are accessible in service workers. However, Local Storage and
+Session Storage are not.
+
+Instead, use an [offscreen document][offscreen]. For example, you could use the following to migrate
+data when updating from Manifest V2 to Manifest V3:
+
+1. In the extension service worker check `chrome.storage` for your data.
+1. If your data isn't found, [create][create-offscreen] an offscreen document and call
+[`sendMessage()`][send-message] to start a conversion routine. Read the data from the web storage
+APIs and use message passing to return it to the service worker.
+1. In the service worker, write the data to a new location.
+
+### Partitioning {: #storage-partitioning }
+
+To prevent certain types of cross-site tracking, [storage partitioning][storage-partitioning]
+creates a seperate storage area in certain third-party contexts. In practice, this means that
```

Comment:
I'd lean on the description in the Privacy Sandbox page. This is a change to how the partitioning key is defined, so it's not separate storage in certain third-party contexts - but a double-keyed structure that results in separate storage for any context.